appcli/init_cli.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+# # -*- coding: utf-8 -*-
+
+"""
+Initialises the system.
+________________________________________________________________________________
+
+Created by brightSPARK Labs
+www.brightsparklabs.com
+"""
+
+# vendor libraries
+import click
+
+# our library
+from .models import Configuration
+from .bsl_keycloak import Keycloak
+
+# ------------------------------------------------------------------------------
+# CONSTANTS
+# ------------------------------------------------------------------------------
+
+# ------------------------------------------------------------------------------
+# CLASSES
+# ------------------------------------------------------------------------------
+
+class InitCli:
+
+    def __init__(self, configuration: Configuration):
+        self.app_name = configuration.app_name
+
+        # ------------------------------------------------------------------------------
+        # CLI METHODS
+        # ------------------------------------------------------------------------------
+
+        @click.group(invoke_without_command=True, help='Initialises the application.')
+        @click.pass_context
+        def init(ctx):
+            if not ctx.invoked_subcommand is None:
+                # subcommand provided, do not enter interactive mode
```

Comment:
remove `do not enter interative mode` (it's confusing here as there is no interative mode)